src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortController.md

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
@kareemmyk  I dont see the reference of AbortController and AbortSignal in Changelog. Is this intended for UXPv7.0?

Reply:
@padmkris123  .... The APIs for AbortController and AbortSignal was added long back.
But we added some documentation for those APIs in the current release.
Hence, it's not added into the changelog.